web_src/js/components/RepoBranchTagSelector.vue

```diff
@@ -246,7 +246,27 @@ const sfc = {
         event.preventDefault();
         this.menuVisible = false;
       }
-    }
+    },
+    fetchBranchesOrTags: onInputDebounce(async function() {
+      const resp = await fetch(`${this.repoLink}/${this.mode}/list`);
+      const {results} = await resp.json();
```

Comment:
Should we perhaps cache the result?
So that we only query it once instead of whenever the user opens the selector?

Reply:
I guess if we do caching, we should do it in a fetch wrapper shared for all fetches, and likely opt-in per consumer.